api/presenters/PostPresenter.js

```diff
@@ -62,10 +64,17 @@ var postAttributes = (post, opts) => {
       public: post.get('visibility') === Post.Visibility.PUBLIC_READABLE
     })
   if (opts.withComments) {
-    console.log('COMMENTS', rel.comments)
+    extendedPost.comments = rel.comments.map(c => _.merge(
+        c.pick('id', 'text', 'created_at', 'user'),
+        {user: c.relations.user.pick('id', 'name', 'avatar_url')}
+      ))
   }
   if (opts.withVotes) {
-    console.log('VOTES', rel.votes)
+    extendedPost.voters = rel.votes.map(v => v.relations.user.pick('id', 'name', 'avatar_url'))
+
+    // recalculate myVote
+    // (the above calculation relies on rel.votes only containing the current users vote, which is not true if opt.withVotes)
+    extendedPost.myVote = _.includes(extendedPost.voters.map(v => v.id), userId)
```

Comment:
can you drop `votes` from the response here so that the new frontend code has to depend upon `voters.length`? then we can drop the nonstandard property once we phase out the old frontend